_compdemos/rslurm_example.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Rslurm and Tximport Example
+date: "5/30/2019"
+main_author: Jenny Smith
+primary_reviewers: vortexing, atombaby
+---
+
+This is an example of Rslurm for a bioinformatics application with the `tximport` package for reading in transcript level RNA-seq count data. I used this example with `tximport` since transcript level counts files, at least for kallisto, are text files that are created for each individual sample. So, if you wanted to combine 100's or 1000's of samples quantification results, the RAM usage can become larger than your local compute requirements.
+
+The takehome message for this example is really on RSlurm. The idea is that one will be able to run an interactive R session and for any large computations/ computationally heavy functions you can simply send that to the HPC and you can continue working in your Rsession while the computation runs on the cluster, thus avoiding the bottle neck in an interactive R session where you have to wait for a long function to finish running.
+
+
+## Purpose
+- Be able to submit analyses in R to the Gizmo HPC cluster
+- Gain familiarity with Rslurm package
+- Gain familiarity with Tximport package
+
+## Audience
+- Those with some knowledge of using `sbatch` to submit jobs to the HPC
+- Some R programming experience
+- Familiarity running Rstudio or R on the Gizmos/Rhinos
+
+If you'd like to know more about using `sbatch` on the command line, see [this page](https://github.com/FredHutch/slurm-examples/tree/master/R_and_sh_example)  and [this page](https://sciwiki.fredhutch.org/scicomputing/compute_jobs/) for more information and some templates. Common slurm commands can also be found in the official SLURM [documentation](https://slurm.schedmd.com/sbatch.html).
+
+## Requirements
+- Will need a small amount of scratch space in your working directory to create dummy expression files and save the output of rslurm
+- You will need access to the Rhinos for submitting the SBATCH job (otherwise, you can just skip that step)
+
+
+More information on `Rslurm` is on its [vignette](https://cran.r-project.org/web/packages/rslurm/vignettes/rslurm.html) and `tximport` also has a nice [vignette](https://bioconductor.org/packages/devel/bioc/vignettes/tximport/inst/doc/tximport.html).
+
+
+# Instructions on how to get set-up
+
+1. open a terminal
+2. ssh user@rhino
```

Comment:
If X11 forwarding is needed, should we do: ssh -X -Y user@rhino?

Reply:
No, as a rule one or the other is sufficient.  @vortexing has opened an issue for us (scicomp) to figure out which to recommend.